seed/data_importer/tests/integration/test_data_import.py

```diff
@@ -389,3 +389,92 @@ def import_exported_data(self, filename):
+        from seed.lib.mcm.reader import ExcelParser
+        col_addr, col_postal = 0, 0
+        self.parser = ExcelParser(self.import_file.file, 'sheet1')
+        for col in range(self.parser.sheet.ncols):
+            if self.parser.sheet.cell_value(0, col) == 'address line 1':
+                col_addr = col
+            elif self.parser.sheet.cell_value(0, col) == 'postal code':
+                col_postal = col
+
+        # get mapped property postal_code
+        ps = PropertyState.objects.filter(address_line_1='11 Ninth Street')[0]
+
+        # regular or w/ ext case:
+        for row in range(self.parser.sheet.nrows):
+            if self.parser.sheet.cell(row, col_addr).value == '11 Ninth Street':
+                if '-' in ps.postal_code:
+                    self.assertEqual(len(ps.postal_code.split('-')[0]), 5)
+                    self.assertEqual(len(ps.postal_code.split('-')[1]), 4)
+                    self.assertEqual(ps.postal_code.split('-')[0].lstrip('0'),
+                                     self.parser.sheet.cell_value(row, col_postal).split('-')[0])
+                    self.assertEqual(ps.postal_code.split('-')[1].lstrip('0'),
+                                     self.parser.sheet.cell_value(row, col_postal).split('-')[1])
+                else:
+                    self.assertEqual(len(ps.postal_code), 5)
+                    self.assertEqual(int(ps.postal_code.lstrip('0')),
+                                     int(self.parser.sheet.cell_value(row, col_postal)))
+
+    def test_postal_code_taxlot(self):
+
+        new_mappings = copy.deepcopy(self.fake_mappings['taxlot'])
+
+        tasks.save_raw_data(self.import_file.pk)
+        Column.create_mappings(new_mappings, self.org, self.user, self.import_file.pk)
+        tasks.map_data(self.import_file.pk)
+
+        # get excel file raw postal code cell value
+        from seed.lib.mcm.reader import ExcelParser
+        col_addr, col_postal = 0, 0
+        self.parser = ExcelParser(self.import_file.file, 'sheet1')
+        for col in range(self.parser.sheet.ncols):
+            if self.parser.sheet.cell_value(0, col) == 'address':
+                col_addr = col
+            elif self.parser.sheet.cell_value(0, col) == 'postal code':
+                col_postal = col
+
+        # get mapped taxlot postal_code
+        ts = TaxLotState.objects.filter(address_line_1='93029 Wellington Blvd').first()
+        if ts is None:
+            raise TypeError("Invalid Taxlot Address!")
+
+        # regular or w/ ext case:
+        for row in range(self.parser.sheet.nrows):
+            if self.parser.sheet.cell(row, col_addr).value == '93029 Wellington Blvd':
+                if '-' in ts.postal_code:
+                    self.assertEqual(len(ts.postal_code.split('-')[0]), 5)
+                    self.assertEqual(len(ts.postal_code.split('-')[1]), 4)
+                    self.assertEqual(ts.postal_code.split('-')[0].lstrip('0'),
```

Comment:
making these explicit checks would be nice, (i.e., something like `self.assertEqual(ts.postal_code.split('-')[0].lstrip('0'), '05005')`)

Reply:
I can change that. Was thinking about doing a general format check instead of spot checking.

Reply:
For whether '0' zip should stay '0' or be packed with 0s, my reason of packing it with 0s is to show users that this is supposed to be a zip code but it's 0. Although I'm not sure if this is a better idea than leaving it as it is which is just a single digit 0.

Reply:
Yeah, I think we want this to be packed, that is 0 -> 00000. Right now it doesn't appear to do that.